paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheetConfigurationValidator.kt

```diff
@@ -0,0 +1,21 @@
+package com.stripe.android.paymentsheet
+
+import java.security.InvalidParameterException
+
+internal fun PaymentSheet.Configuration.validate() {
+    // These are not localized as they are not intended to be displayed to a user.
+    when {
+        merchantDisplayName.isBlank() -> {
+            throw InvalidParameterException("Merchant display name cannot be blank.")
+        }
+        customer?.id?.isBlank() == true -> {
+            throw InvalidParameterException("Customer id when specified cannot be blank.")
```

Comment:
`When the customer is specified the id cannot be blank.`

Reply:
nit but "Customer" as we're referring to capital-c Customer API primitive. 

And same comment as below around usage of "blank" vs "empty string"